apps/dapp/src/components/Pages/Core/DappPages/Dashboard/hooks/use-dashboardv2-metrics.ts

```diff
@@ -1,16 +1,8 @@
-import { useQuery } from '@tanstack/react-query';
 import millify from 'millify';
 import { fetchGenericSubgraph, fetchSubgraph } from 'utils/subgraph';
-import { DashboardType } from '../DashboardContent';
 import env from 'constants/env';
 import { useMemo } from 'react';
-
-export enum StrategyKey {
-  RAMOS = 'RamosStrategy',
-  TLC = 'TlcStrategy',
-  TEMPLEBASE = 'TempleBaseStrategy',
-  DSRBASE = 'DsrBaseStrategy',
-}
+import { getQueryKey, StrategyKey, useApiQuery } from 'hooks/api/use-react-query';
```

Comment:
My preference for this file, since I'm still actively working on it, would be to keep it as-is, just to avoid a messy merge conflict. Then, after you merge, we can bring many of these changes into this file.

So basically just rollback all the changes except the generics, wdyt?

Reply:
I'll revert back the changes for this file 👍